app/helpers/recordings_helper.rb

```diff
@@ -36,7 +36,7 @@ def recording_length(playbacks)
     elsif len.zero?
       "< 1 min"
     else
-      "#{len} min"
+      "#{len} mins"
```

Comment:
This should be localized, is it?

Reply:
Yes correct. I'm going to remove it from this PR and create another PR to fix it and other localization issues